manifests/c/ca/duan/tre-command/0.4.0/ca.duan.tre-command.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ca.duan.tre-command
+PackageVersion: 0.4.0
+PackageLocale: en-US
+Publisher: Daniel Duan
+PackageName: tre

```

```suggestion
PackageName: tre-command
```
This must match Control Panel.